src/lib/probe.c

```diff
@@ -87,6 +175,14 @@ CbmDeviceProbe *cbm_probe_path(const char *path)
                 }
         }
 
+        /* Ensure that this parent disk really is LUKS */
+        if (blkid_probe_lookup_value(blk_probe, "TYPE", &value, NULL) != 0) {
```

Comment:
I thought that cbm_probe_path was providing the part_uuid for normal use as well in addition to the luks path. And the type would have been NULL in the non luks existance case.